modules/tensor_mechanics/test/tests/2D_different_planes/gps_xz.i

```diff
@@ -0,0 +1,214 @@
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Mesh]
+  file = square_xz_plane.e
```

Comment:
How different is this test from the other? Just a different plane? Perhaps you can reuse the first input file instead. We have mesh transformations which can transform the mesh to a different plan and then a few other minor changes could be made. This keeps down on duplication, simplifies maintenance.

Reply:
I was thinking of just using cli args and the first input file to test different planes.  However, from what I can see I can't change variable names  or anything in [./name] via the command line arguments. With the number of places the variables show up and need to be changed it seemed to warrant a different input file.

Reply:
OK - seems reasonable... 

There are other options though. You could actually use variable expansion syntax and change the name in one place, or use a generic name instead of one relating to the direction. Just ideas to keep duplication down in the future.